packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -130,8 +131,35 @@ export async function runInterstitialRules<T>(opts: T, rules: InterstitialRule[]
 async function verifyRequestState(options: any, token: string) {
   const issuer = (iss: string) => iss.startsWith('https://clerk.') || iss.includes('.clerk.accounts');
 
+  if (options?.isSatellite) {
+    return verifyToken(token, {
+      ...options,
+      issuer: null,
+    });
+  }
+
   return verifyToken(token, {
     ...options,
     issuer: options?.proxyUrl || issuer,
   });
 }
+
+export const isSatelliteAndNeedsSync: InterstitialRule = async options => {
+  const { clientUat, isSatellite, isSynced = false } = options as AuthenticateRequestOptionsWithExperimental;
+
+  if (isSatellite && (!clientUat || clientUat === '0') && !isSynced) {
+    return interstitial(options, AuthErrorReason.SatelliteCookieNeedsSync);
+  }
+
+  return undefined;
+};
+
+export const isSatelliteAlreadySyncedButCookieIsMissing: InterstitialRule = async options => {
+  const { clientUat, cookieToken, isSatellite, isSynced } = options as AuthenticateRequestOptionsWithExperimental;
+
+  if (clientUat && Number.parseInt(clientUat) > 0 && !cookieToken && isSatellite && isSynced) {
+    return interstitial(options, AuthErrorReason.SatelliteCookieMissing);
+  }
+
+  return undefined;
+};
```

Comment:
🔧  The conditions of those 2 rules are not in sync with the flow diagram. We should update the flow diagram or the conditions.